src/components/icon/icon.tsx

```diff
@@ -82,6 +88,9 @@ export class Icon {
 
   componentWillLoad() {
     this.inheritedAttributes = inheritAttributes(this.el, ['aria-label']);
+    this.shouldAutoFlip =
+      (this.iconName?.includes('arrow') || this.iconName?.includes('chevron')) && this.flipRtl !== false;
+    this.shouldBeFlippable = this.flipRtl || this.shouldAutoFlip || this.el.hasAttribute('dir');
```

Comment:
This should probably stay in the `render` function in the event that the type of icon changes after page load. (Silly example: icon changes from `alarm` to `arrow-down`)